src/frontends/ir/src/ir_deserializer.cpp

```diff
@@ -596,10 +598,21 @@ class MetaDataParser : public ov::Meta {
     }
 
     operator ov::AnyMap&() override {
+        m_accessible_by_pugixml_node = false;
```

Comment:
Maybe the `m_meta_node` should be set in ctor depends on bool flag. and here should be changed as bool flag will be always false. Then `get_pugi_node` can return reference without any action.

Reply:
I set m_meta_node in ctor and change the check in get_pugi_node method. So now assignment is called only once + atomic_bool is loaded and stored also only once